src/pages/version/taskDuration/TaskDurationTable.tsx

```diff
@@ -0,0 +1,279 @@
+import styled from "@emotion/styled";
```

Comment:
Sorry this file is still not great, I don't know if it's possible to simplify any further given the restrictions with the Table.